exercises/D-logical-operators/exercise.js

```diff
@@ -14,12 +14,36 @@ let cssLevel = 4;
 let htmlLevelAbove5;
 let cssLevelAbove5;
 
+if (htmlLevel > 5) {
+  htmlLevelAbove5 = true;
+} else {
+  htmlLevelAbove5 = false;
+}
+
+if (cssLevel > 5) {
+  cssLevelAbove5 = true;
+} else {
+  cssLevelAbove5 = false;
+}
+
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
 let cssAndHtmlAbove5;
 let cssOrHtmlAbove5;
 
+
+if (htmlLevel > 5 && cssLevelAbove5 > 5){
+  cssAndHtmlAbove5 = true;
+}else {
+  cssAndHtmlAbove5 = false;
+}
+
+if (htmlLevel > 5 || cssLevelAbove5 > 5){
+  cssOrHtmlAbove5 = true;
+}else {
+  cssOrHtmlAbove5 = false;
+}
```

Comment:
This also works well, and you have understood how to us AND and OR. Again, you could assigning the result of the comparison directly to the variables.

Reply:
Done